src/components/clusterWizard/ClusterWizardContext.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { ClusterWizardStepsType, CLUSTER_WIZARD_FIRST_STEP } from './wizardTransition';
+import { ClusterWizardStepsType, getClusterWizardFirstStep } from './wizardTransition';
 
 type ClusterWizardContextType = {
   currentStepId: ClusterWizardStepsType;
   setCurrentStepId: (stepId: ClusterWizardStepsType) => void;
 };
 
 const ClusterWizardContext = React.createContext<ClusterWizardContextType>({
-  currentStepId: CLUSTER_WIZARD_FIRST_STEP,
+  currentStepId: getClusterWizardFirstStep(),
```

Comment:
IIUC we could put just 'cluster-details' here as this value is only used in case when context provider is not rendered.